content/project-organization/people-and-roles.md

```diff
@@ -98,47 +103,28 @@ Alumni:
 - Antonin Clauzier ([@0x346e3730](https://github.com/0x346e3730))
 - Julius Žukauskas ([@zuk3975](https://github.com/zuk3975))
 - Mahmoud Parvazi ([@mparvazi](https://github.com/mparvazi))
+- Marco Salvatore ([@marsaldev](https://github.com/marsaldev))
+- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
+- Louis Authie ([@LouisAUTHIE](https://github.com/LouisAUTHIE))
 - Salim Benouamer ([@lartist](https://github.com/lartist))
 - Thomas Baccelli ([@atomiix](https://github.com/atomiix))
 
 ## Quality Council
 
 The quality council provides the quality assurance vision for the project and has authority on related subjects, such as verification of pull requests, test automation strategy, bug triage process, and quality assessment.
 
-### Issue Managers
```

Comment:
Removed this team with approval of @Robin-Fischer-PS as it was duplicate of "Software Testers" team